Add plugin: Blueprint

madx opened the pull request:
# I am submitting a new Community Plugin

- [x] I attest that I have done my best to deliver a high-quality plugin, am proud of the code I have written, and would recommend it to others. I commit to maintaining the plugin and being responsive to bug reports. If I am no longer able to maintain it, I will make reasonable efforts to find a successor maintainer or withdraw the plugin from the directory.

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/madx/blueprint-obsidian-plugin/

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [x]  Linux
  - [x]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

Fevol commented:
## Required Changes

Your plugin description should follow [the style guide](https://docs.obsidian.md/Plugins/Releasing/Submission+requirements+for+plugins#Keep+plugin+descriptions+short+and+simple), i.e. the description should end with a period.

[Prefer `Vault.process` instead of `Vault.modify` to modify a file in the background.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Prefer+%60Vault.process%60+instead+of+%60Vault.modify%60+to+modify+a+file+in+the+background)
https://github.com/madx/blueprint-obsidian-plugin/blob/df1094ddb205c1f752413cd259c10d4d607da9b0/src/commands.ts#L52


---

## Optional Feedback
N/A

madx commented:
> https://github.com/tutivsoft-com/obsidian-denali-ai-file-renamer-front-matter/blob/120653d1e3b6988fccca0e0c243e078362502f6e/main.ts#L1709 Your license should be updated to this year, sorry for the wait!😅

Fixed in https://github.com/madx/blueprint-obsidian-plugin/commit/b39fd6980f232b23fad54cf64fd5287098ce2216

> https://github.com/madx/blueprint-obsidian-plugin/blob/main/README.md#L1 https://github.com/madx/blueprint-obsidian-plugin/blob/933b72d9fcbfbecb5e1aff4ec3b1fbc88bd6ef6d/CHANGELOG.md#L3 You may want to consider updating your README and CHANGELOG to include the new changes you have added to the plugin.

Fixed in https://github.com/madx/blueprint-obsidian-plugin/commit/82be04b5e52e721fcf90903ceb8d6ab409308cc7

Thanks for your review!
